### PR TITLE
korenka: Updated create instance signal response

### PR DESCRIPTION
**Pull request by korenka**
AWSEG-25392

_This is the default pull request template. You can customize it by adding a `pull_request_template.md` at the root of your repo or inside the `.github` folder._

# Jira Ticket

https://spotinst.atlassian.net/browse/AWSEG-25392

# Demo

_Please add a recording of the feature/bug fix in work. if you added new routes, the recording should show the request and response for each new/changed route_

# Checklist:
- [ ] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [ ] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have validated all the requirements in the Jira task were answered
- [ ] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**)

### CODE REVIEW

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml [hunk 1]
```diff
@@ -48,4 +48,45 @@ content:
...
+                            instanceLaunchedAt:
+                              type: string
+                              example: 2023-04-30T12:20:14.000Z
+                              description: >
+                                The date and time that the instance has been launched at (Present only if INSTANCE_READY signal is configured for the group)
+                            instanceDetachedAt:
+                              type: string
+                              example: 2023-04-30T12:20:14.000Z
+                              description: >
+                                The date and time that the instance has been detached at (Present only if INSTANCE_READY_TO_SHUTDOWN signal is configured for the group)
```
Comment (idohas) [inline api/services/elastigroup/aws/responses/create-instance-signal.yaml:74]:
```suggestion
                              description: >
                                The time the instance was detached (returned only for signals of type INSTANCE_READY_TO_SHUTDOWN).
```

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml [hunk 2]
```diff
@@ -48,4 +48,45 @@ content:
...
+                            type:
+                              type: string
+                              example: INSTANCE_READY
+                              description: >
+                                The type of signal sent in the request
+                            signalExpiresAt:
+                              type: string
+                              example: 2023-04-30T12:21:14.000Z
+                              description: >
+                                The date and time that the signal will be expired at
```
Comment (idohas) [inline api/services/elastigroup/aws/responses/create-instance-signal.yaml:84]:
```suggestion
                                The time at which the signal expires.
```

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml [hunk 3]
```diff
@@ -48,4 +48,45 @@ content:
...
+                            message:
+                              type: string
+                              example: This instance crossed the signal timeout limit at Sun Apr 30 12:21:14 UTC 2023
+                              description: >
+                                If the instance has crossed the signal timeout limit - this message will be sent in the response
```
Comment (idohas) [inline api/services/elastigroup/aws/responses/create-instance-signal.yaml:89]:
```suggestion
                                An optional message that may contain additional information.
```

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml [hunk 4]
```diff
@@ -48,4 +48,45 @@ content:
...
+                        kind:
+                          type: string
+                          example: spotinst:aws:ec2:instance:signal
+                          description: >
+                            The data type of each item in the response
+                        items:
+                          type: object
+                          properties:
+                            instanceId:
+                              type: integer
```
[the remark this answers is not in the document]
Comment (korenka) [reply api/services/elastigroup/aws/responses/create-instance-signal.yaml:61]: done

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml [hunk 5]
```diff
@@ -48,4 +48,45 @@ content:
...
+                              example: i-0c8bc7fd0e8cd6c8d
+                              description: >
+                                The instance ID that is configured with signals
```
[the remark this answers is not in the document]
Comment (korenka) [reply api/services/elastigroup/aws/responses/create-instance-signal.yaml:64]: done

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml [hunk 6]
```diff
@@ -48,4 +48,45 @@ content:
...
+                            instanceLaunchedAt:
+                              type: string
+                              example: 2023-04-30T12:20:14.000Z
+                              description: >
+                                The date and time that the instance has been launched at (Present only if INSTANCE_READY signal is configured for the group)
```
[the remark this answers is not in the document]
Comment (korenka) [reply api/services/elastigroup/aws/responses/create-instance-signal.yaml:69]: done

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml (hunk 1 above)
Comment (korenka) [reply api/services/elastigroup/aws/responses/create-instance-signal.yaml:74]: done

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml [hunk 7]
```diff
@@ -48,4 +48,45 @@ content:
...
+                            instanceDetachedAt:
+                              type: string
+                              example: 2023-04-30T12:20:14.000Z
+                              description: >
+                                The date and time that the instance has been detached at (Present only if INSTANCE_READY_TO_SHUTDOWN signal is configured for the group)
+                            type:
+                              type: string
+                              example: INSTANCE_READY
+                              description: >
+                                The type of signal sent in the request
```
[the remark this answers is not in the document]
Comment (korenka) [reply api/services/elastigroup/aws/responses/create-instance-signal.yaml:79]: done

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml (hunk 2 above)
Comment (korenka) [reply api/services/elastigroup/aws/responses/create-instance-signal.yaml:84]: done

File: api/services/elastigroup/aws/responses/create-instance-signal.yaml (hunk 3 above)
Comment (korenka) [reply api/services/elastigroup/aws/responses/create-instance-signal.yaml:89]: done

**Review by idohas — APPROVED** [review]

**Pull request merged**